crates/test/macros/src/lib.rs

```diff
@@ -124,15 +124,16 @@ fn tests_runner_impl(args: TokenStream) -> parse::Result<TokenStream> {
                 ret
             }
 
-            fn test_list() -> impl Iterator<Item = (&'static str, &'static str, ::flipperzero_test::TestFn)> {
+            fn test_list() -> impl Iterator<Item = (&'static str, &'static str, ::flipperzero_test::TestFn)> + Clone {
```

Comment:
Since we are in a macro context here, I think that we should use fully qualified path to `Iterator` and `Clone`

Reply:
I don't believe that to be necessary, as both of these are part of [`core::prelude`](https://doc.rust-lang.org/core/prelude/v1/index.html), and are therefore available directly in every `no-std` Rust program.

Reply:
AFAIK, there are at least two ways to break this i.e.:
* `#![no_implicit_prelude]`
* having similarly named traits in the scope.

While the former is quite exotic, the latter may appear thus I would prefer having a bit more hygienic fix for this.

Arguing against myself, it still is not 100%-hygienic, but to me it looks like a more error-prone solution.

Reply:
Let's address this in a separate PR then, that goes through all the macros and ensure they follow a consistent pattern.